.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions? Did you do something clever?
+
```

Comment:
Add a line here to remind of backward compatibility concerns, like so,

"Is the change backward compatible? Or, are there concerns around backward compatibility?"

Reply:
Done.